### DISCUSSION TITLE
Save a lossless png (with compression 0) to preserve quality for post-processing

### DISCUSSION THREAD

**Discussion by CapNemo88** [comfyanonymous/ComfyUI#567] [Q&A] [answered]
Does anybody knows whether it is possible to save output pics in png with compression 0?
It would be nice to have max quality for post-processing...
Any hints how to change configs to achieve that, please?

**Comment by ltdrdata** [comment, marked as answer]
PNG is already lossless. Why do you need compression level adjustment.

**Reply by CapNemo88** [reply]
Sorry for a dumb question. You are right. I needed to google for that, sorry.
Thank you.
